packages/plasma-new-hope/src/components/Modal/Modal.tsx

```diff
@@ -42,7 +43,7 @@ export const modalRoot = (Root: RootProps<HTMLDivElement, ModalProps>) =>
             },
             outerRootRef,
         ) => {
-            const trapRef = useFocusTrap(true, initialFocusRef, focusAfterRef);
+            const trapRef = useFocusTrap(true, initialFocusRef, focusAfterRef, true);
```

Comment:
Поскольку у хука в связи с правками стало четыре аргумента, то сигнатура стала менее явной. Раз выносим хук из `core`, то может соберём аргументы в объект, чтобы в будущем было легче использовать?

```suggestion
            const trapRef = useFocusTrap({ active: true, firstFocusSelector: initialFocusRef, focusAfterNode: focusAfterRef, activeAfterAnimation: true };
```

Reply:
хук используется много где, должна быть совместимость

Reply:
ок, если в будущем будет полный перенос хука из `core` в `new-hope` можно подумать